apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -131,6 +131,27 @@ window.PLOrderBlocks = function (uuid, options) {
       placePairingIndicators();
       setAnswer();
     },
+    start: function () {
+      if (enableIndentation) {
```

Comment:
I would prefer you take all this code and put it in its own function with a name, so that people reading the codebase know what the code does without having to read it all. Call it something helpeful like `function drawIndentLocationLines` or something.

Reply:
Done, also added some comments to the grid line drawing code.